### PR TITLE
Example for intersperse

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ Accumulate incoming events until the specified number of elements have been accu
 Accumulate incoming events until the specified number of elements have been accumulated and then pass the collection of
 elements downstream.
 
+## Examples
```
Comment (raboof) [inline akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md]: Did we decide on having the examples above the RS semantics?

I don't remember and we're not doing it consistently now - happy to help make it consistent if this is what we decided on ;)

**Review by raboof — APPROVED** [review]
Short and sweet ;)

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md (hunk 1 above)
Comment (johanandren) [reply akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md]: I think we did decide on that

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md (hunk 1 above)
Comment (raboof) [reply akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md]: :+1: I'll PR that then

Comment (raboof) [reply akka-docs/src/main/paradox/stream/operators/Source-or-Flow/grouped.md]: https://github.com/akka/akka/pull/28357

**Pull request merged**